SECURITY.md

```diff
@@ -0,0 +1,5 @@
+## Reporting a security issue
+
+If you find any security issue or vulnerability, please email us at INSERT EMAIL HERE with your report.
```

Comment:
@ben Please make a suggestion for the correct email adress, as I don't know which email you want to use. 😉